api/state.go

```diff
@@ -210,14 +210,7 @@ func (st *state) setLoginResult(p loginResultParams) error {
 	st.controllerAccess = p.controllerAccess
 	st.modelAccess = p.modelAccess
 
-	hostPorts, err := addAddress(p.servers, st.addr)
-	if err != nil {
-		if clerr := st.Close(); clerr != nil {
-			err = errors.Annotatef(err, "error closing state: %v", clerr)
-		}
-		return err
-	}
-	st.hostPorts = hostPorts
```

Comment:
I have fixed the PR so that we are only ignoring the socket address when there is a proxy case involved (juju internal proxy).

We are doing a seperate PR to make the python lib understand the proxy information.